tests/test_strds_area_stats.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
 import unittest
-import time
-from pprint import pprint
-from flask.json import loads as json_load
-from flask.json import dumps as json_dump
-
-try:
-    from .test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
-except:
-    from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
+# import time
+# from pprint import pprint
+# from flask.json import loads as json_load
+# from flask.json import dumps as json_dump
+#
+# try:
+#     from .test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
+# except Exception:
+#     from test_resource_base import ActiniaResourceTestCaseBase, URL_PREFIX
```

Comment:
Is it on purpose that these imports and the imports in the following are commented out now?

Reply:
they area commented out, because the tests are not running yet and so they are also outcommented. In another MR I will try to fix the tests.